### ISSUE TITLE
Hashtags are displayed all uppercase

### ISSUE THREAD

**Issue by gunchleoc** [Chocobozzz/PeerTube#7076]
### Describe the current behavior

Hashtags that used to be given & displayed as camelCase or PascalCase are displayed in ALLUPPERCASE since version `7.2.0`. 

This is a problem for dyslexics and people who depend on screenreaders, which is why I am considering this a regression bug, in spite it maybe being an intentional redesign.

### Steps to reproduce

1. Manage a video
2. Type a tag in camelCase or PascalCase
3. Hit Enter

### Describe the expected behavior

Display tags the way they were typed in.

### Additional information

* PeerTube instance:
  * URL: https://tbh.co-shaoghal.net
  * Version: 7.2.0
  * NodeJS version: v20.19.2
  * Ffmpeg version: 5.1.6-0+deb12u1 

* Browser name, version and platforms on which you could reproduce the bug: Firefox 139.0.1, Linux Mint 21.3 Virginia 64-bit

**Comment by gunchleoc** [comment]
While you're touching this, https://github.com/Chocobozzz/PeerTube/issues/5017 might also be of interest.

**Comment by Chocobozzz** [comment]
Thanks! It should be fixed by https://github.com/Chocobozzz/PeerTube/commit/11f769eadeae6f9e6e3b82c06dc7f8b7621a5278
